pkg/config/config_template.yaml

```diff
@@ -1061,7 +1061,9 @@ api_key:
   ## @env DD_APM_MAX_TPS - integer - optional - default: 10
   ## The target traces per second to sample. Sampling rates to apply are adjusted given
   ## the received traffic and communicated to tracers. This configures head base sampling.
-  ## Set to 0 to disable sampling.
+  ## As of 7.35.0 setting max_traces_per_second to 0 no longer disables sampling and will instead
+  ## send no traces to the intake. Sampling can not be disabled, to avoid rate limiting set this value
```

Comment:
```suggestion
  ## As of 7.35.0 setting max_traces_per_second to 0 no longer disables sampling and instead
  ## sends no traces to the intake. Sampling can not be disabled, to avoid rate limiting set this value
```

Reply:
Thanks for the PR, from a docs team perspective the other edits mentioned here look good, but I just wanted to make one small addition per our contributing guidelines:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense